centreon/www/include/configuration/configKnowledge/functions.php

```diff
@@ -11,6 +11,32 @@
  *
  */
 
+/**
+ * This returns an anonymous class to manage alternate CSS class for table lines TR.
+ */
+function getLineTemplate(string $class1, string $class2): object
```

Comment:
Maybe rename the $class1 and $class2 variable , it could be confusing between a PHP Class and a CSS class